.github/scripts/run_ci.sh

```diff
@@ -165,6 +165,10 @@ fi
 # publish to GitHub. It helps finding out any issue in pull requests and dev
 # branch.
 # =============================================================================
+echo "Step: Bundling default plugins in desktop application"
+cd "$ROOT_DIR/packages/tools"
+tsc bundleDefaultPlugins.ts
+node bundleDefaultPlugins.js
```

Comment:
This code is only necessary when building the desktop app, but you have it here where it will be called even when building the server image. I think this code only needs to be run for the first block below (Building the desktop application for publishing).

Reply:
I will move it then. I thought Docker Image also needs these plugins.